index.md

```diff
@@ -18,10 +18,31 @@ Please submit a pull request if you have something to add!
 
 * [Stephen Holiday](https://stephenholiday.com)
 * [Tony Dong](tony-dong.com)
+* [Michael Shao](http://michaelshao.com)
 
 <section id="Getting-the-Job" />
 ## Getting the Job
 
+<section id="Applying-to-Jobs" />
+### Applying to Jobs
+
+If you're a graduating co-op student, attending Waterloo or elsewhere, you already know how this works, for the most part.
+Here are a few quick resource refreshers for finding awesome companies to work at.
+
+* [AngelList](https://angel.co) - AngelList has a wealth of early to late-stage (well-funded) startups that are always looking for talent. Post a talent profile and search for that full-time job.
+* [LinkedIn Premium](https://premium.linkedin.com) - LinkedIn itself already has some great tools; adding a premium search functionality to that experience can only enhance your search.
```

Comment:
Yeah, both of them got me a lot of interviews, since I didn't take any of my return offers from co-op and had trouble finding interviews inside of JobMine alone.

I was able to get an offer each from each path - AngelList got me an offer from a startup in SF, and LinkedIn Premium got me an offer from Cisco in Ottawa.